### ISSUE TITLE
question during reproducing results

### ISSUE THREAD

**Issue by Junwen-Huang** [TachibanaYoshino/AnimeGAN#31]
Great job! 
I ran the code with python 3.7.4 and tf 1.15.0 to reproducing the results. And I found it hard to train the Paprika style well with default hyperparameter settings. It seems that the GAN just transforms the image close to grayscale, and there are also some vertical lines.
![006_a](https://user-images.githubusercontent.com/14028928/84100223-ffd38280-aa3d-11ea-9cd8-79a60fa08895.png)
![006_b](https://user-images.githubusercontent.com/14028928/84100230-0235dc80-aa3e-11ea-8e8d-64003d45e864.png)
Is it caused by Paprika data?
Note that I trained 130 epoches. The results after 100th epoch looks better, but the green region still goes grayscale.
![006_b (1)](https://user-images.githubusercontent.com/14028928/84100581-c5b6b080-aa3e-11ea-8df4-209c67f45987.png)
It means that I just need to select a proper checkpoint during training? If yes, how?

**Comment by TachibanaYoshino** [comment]
If you want to train more epochs, you need to reduce the learning rate. In fact, according to the effect of different epoch verification sets, choose the best model from them. For red pepper style, you can adjust the style-weight to between 1.1-1.3. The data style is generally dim. If the saturation is not enough, the loss weight of the U and V channels can be increased appropriately.

**Comment by TachibanaYoshino** [comment]
By the way, the Paprika  style data needs to be reworked, and the data quality is not high enough.

**Comment by Junwen-Huang** [comment]
Thanks! Practical suggestion. Since the repo is updated, I will try it again.

**Comment by tankfly2014** [comment]
yes. no easy way.

I try run example point file AnimeGAN.model-60.meta. from epoch=60.  start run train.
d_loss show start value 2. gogogo, My train result d_loss>2, =20.  The number d_loss has high and the situation has deteriorated.

I just failed to use the default parameters. Adjustments must be made at any time.
If this program wants to be great, the d-loss must be maintained at least to <n? 5? 7?
